src/web/default/views/donate/donate.cshtml

```diff
@@ -30,7 +30,7 @@
                 You can directly donate to our following addresses;<br /><br />
                 <ul>
                     <li>Bitcoin: <a href="bitcoin:1MMdwRZg4K9p6oH2bWUQdohbxGbAvXS1t1?label=CoiniumServ%20Donation"><code>1MMdwRZg4K9p6oH2bWUQdohbxGbAvXS1t1</code></a></li>
-                    <li>Litecoin: <a href="ethereum:Ld8cy4ucf3FYThtfTnRQFFp5MKK9rZHjNg?label=CoiniumServ%20Donation"><code>0x61aa3e0709e20bcb4aedc2607d4070f1db72e69b</code></a></li>
+                    <li>Ethereum: <a href="ethereum:Ld8cy4ucf3FYThtfTnRQFFp5MKK9rZHjNg?label=CoiniumServ%20Donation"><code>0x61aa3e0709e20bcb4aedc2607d4070f1db72e69b</code></a></li>
```

Comment:
@jeong760 

This is still wrong. The eth address in the href does not match the displayed address. It's set to the litecoin address.

Reply:
@jeong760 

It's not an issue if they manually copy and paste. It's an issue if they click on the link.